.env.testnet

```
@@ -2,6 +2,7 @@ CHAIN_ENV=testnet
 
 FUNGIBLE_TOKEN_ADDRESS=0x9a0766d93b6608b7
 NON_FUNGIBLE_TOKEN_ADDRESS=0x631e88ae7f1d7c20
+FUSD_TOKEN_ADDRESS=0xe223d8a629e49c68
```

```suggestion
FUSD_ADDRESS=0xe223d8a629e49c68
```

Just to be consistent with the `FUSD_ADDRESS` variables below